main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:      squirrel
+//  action 1:  hibernates
+squirrel.hibernate();
+//  action 2:  gathers nuts
+squirrel.gatherNuts();
+//  action 3:  runs away
+squirrel.runsAway();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:      cup
+//  action 1:  abides
+cup.abide();
+//  action 2:  shatters
+cup.shatter();
+//  action 3:  conveys liquid
+cup.conveyLiquid();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:      krill
+//  action 1:  drifts
+krill.drift();
+//  action 2:  provides nutrients
+krill.provideNutrients();
+//  action 3:  swim
+krill.swim();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:      mosquito
+//  action 1:  fly
+mosquito.fly();
+//  action 2:  bite
+mosquito.bite();
+//  action 3:  spawn
+mosquito.spawn();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:      SpiderMan
+//  action 1:  swing
+spiderMan.swing();
+//  action 2:  quip
+spiderMan.quip();
+//  action 3:  punch a thug
+spiderMan.punchThug();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:      Seth Rollins
+//  action 1:  Shooting Star Press
+rollins.shootingStarPress();
+//  action 2:  Curb Stomp
+rollins.curbStomp();
+//  action 3:  Pedigree
+rollins.pedigree();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:      Dean Ambrose
+//  action 1:  Dirty Deeds
+ambrose.dirtyDeeds();
+//  action 2:  throw 12-20 chairs
+ambrose.rainChairs();
+//  action 3:  Thesz press
+ambrose.theszPress();
```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise